### DISCUSSION TITLE
Help prettifying broken wikilinks

### DISCUSSION THREAD

**Discussion by kksgandhi** [srid/emanote#220] [Q&A]
At the moment, a broken wikilink has a strikethrough effect, naked brackets, a large X, and a "backlinks" superscript link. This seems rather excessive, and I was wondering if there was a user controllable way to set what broken wikilinks should look like? Furthermore, is there a way to create a template for the "broken link, here's the backlinks" page?

**Comment by srid** [comment]
Perhaps, as you ask, we should let Heist define broken link rendering in `broken.tpl` - so the user can override it entirely. 

Currently the HTML is hardcoded here: 

https://github.com/srid/emanote/blob/e35ce8f0fb9d10262d0c662e18655154a44a4c17/src/Emanote/Pandoc/Renderer/Url.hs#L78-L79

**Reply by kksgandhi** [reply]
I'm rather new to github and this style of collaboration. Would it be reasonable to create an issue / feature request from this discussion?

**Reply by kksgandhi** [reply]
At the moment, I'm thinking of writing a javascript hook that modifies backlinks. Would you recommend this solution for now?

**Reply by srid** [reply]
> At the moment, I'm thinking of writing a javascript hook that modifies backlinks. Would you recommend this solution for now?

Sure. Unless you want to contribute the Haskell code change for #221, a JS based solution would do.

**Reply by kksgandhi** [reply]
Unfortunately I'm not well versed in Haskell
